### DISCUSSION TITLE
PC version

### DISCUSSION THREAD

**Discussion by CelArts** [thatskymod/Sky-CotL-Scripts#66] [Ideas]
Since Sky is now on PC, I'd hella like to see a version of this for PC in the near future!

**Comment by axelnieves2012** [comment]
x2

**Comment by thatskymod** [comment]
Hello,

I believe we can consider this case resolved given that we now have a stable, well-maintained customization for the PC/Steam version. With the new [installer app](https://github.com/XeTrinityz/ThatSkyApp/releases), downloading and deploying the mod should be straightforward.

Credits to @XeTrinityz for making this possible.

Thank you for your contribution!
